spring-rabbit-stream/src/main/java/org/springframework/rabbit/stream/listener/StreamListenerContainer.java

```diff
@@ -117,6 +117,13 @@ public StreamListenerContainer(Environment environment, @Nullable Codec codec) {
 						: new DefaultStreamMessageConverter();
 	}
 
+	/**
+	 * Get a name of stream this listener listens to.
+	 */
+	public String getStreamName() {
+		return streamName;
```

Comment:
Must be with `this.`.
Also add `@since 3.2.3` to this method JavaDocs and your name to the `@author` list.
Thanks.